data/human/human missions.txt

```diff
@@ -5416,3 +5416,63 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+government "Escort (Betraying)"
```

Comment:
```suggestion
government "Escort (Betraying)"
```

I'm not sure, but I think we may able to use the same merchant spoof government here as in the disguised warlord missions?

Reply:
Setting that government to hostile in this mission would break those missions, since it'd mean that in subsequent missions the disguised pirates would start as hostile.